src/services/completions.ts

```diff
@@ -1144,9 +1144,13 @@ namespace ts.Completions {
             }
 
             if (isMetaProperty(node) && (node.keywordToken === SyntaxKind.NewKeyword || node.keywordToken === SyntaxKind.ImportKeyword)) {
-                const completion = (node.keywordToken === SyntaxKind.NewKeyword) ? "target" : "meta";
-                symbols.push(typeChecker.createSymbol(SymbolFlags.Property, escapeLeadingUnderscores(completion)));
-                return;
+                const metaPropertyAccessIdentifier = node.name.text;
+                // Check whether this MetaProperty node already had its metadata accessed
+                if (metaPropertyAccessIdentifier !== "target" && metaPropertyAccessIdentifier !== "meta") {

```

Comment:
Again this is incorrect since even if you have code like: ```import./*completionhere*/meta``` should have meta as completion.

Reply:
I changed it so that the conditional checks whether `MetaProperty`'s `dotToken` is the same one as the `dotToken` bound to `contextToken` variable.

Reply:
I was trying to find a utility function to access `dotToken` but couldn't find any, so I used `getChildAt(1)` instead. But I assume this isn't ideal (It looks like that `getChildAt` function was never used with anything other than `0` as its argument.)